snowflake/tests/keys/rsa_key_example.p8

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCXeqIH2MXr6E7p
```

Comment:
Used example key in `voltdb`
https://github.com/DataDog/integrations-core/blob/119bbccd98f245d02ae3d37492fc7d6c33fe5671/voltdb/tests/compose/certs/client.pem#L5-L32